### ISSUE TITLE
Ignore auto-assign production if "Produce Nothing" was enqueued explicitly

### ISSUE THREAD

**Issue by arbitrary-dev** [yairm210/Unciv#6470]
I have auto-assign production enabled, but want one particular city to "Produce Nothing", so I enqueue it explicitly. Nevertheless on the next turn it's production queue changes to "Worker".

**Comment by alexban011** [comment]
Does this still occur? I couldn't manage to reproduce

**Comment by yairm210** [comment]
Nope, working for me both with and without other items in the queue.
What are the steps to reproduce?
